src/web-ui/src/repositories/usersRepository.js

```diff
@@ -50,9 +50,9 @@ export default {
         if (!userID || userID.length == 0)
             throw "userID required"
         let payload = {
-            userid: userID,
-            phonenumber: phonenumber        
+            user_id: userID,
+            phone_number: phonenumber        
```

Comment:
Naming standard for client side JS is camel case. So `userId` and `phoneNumber` is preferred.

Reply:
Will update.